sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -256,6 +259,33 @@ private boolean canLogAtLevel(LogLevel logLevel, LogLevel environmentLoggingLeve
         }
     }
 
+    /**
+     * Checking the system log with the preference order of slf4j and environment variable.
+     *
+     * @return The log level.
+     */
+    private void loadSystemLogLevel() {
+        if (rootLogLevel != null) {
+            return;
+        }
```

Comment:
This breaks a design guideline where the environment logging level must be able to update at any time, once `rootLogLevel` is set in the `ClientLogger` it will never be able to be changed, I think this should be removed.

https://azure.github.io/azure-sdk/general_implementation.html#general-config-behaviour-changes